src/extension_utils/utils.ts

```diff
@@ -294,6 +294,7 @@ export const promptInstallPythonDependencies = (
     return vscode.window
         .showInformationMessage(
             CONSTANTS.INFO.INSTALL_PYTHON_DEPENDENCIES,
+            { modal: true },
```

Comment:
As we discussed offline, it might be less intrusive to keep without a modal dialog for now? Ideally we wouldn't want that to fade out either, but we are not sure whether that's possible ):

Reply:
Removed the modal styling in the last commit 😁